docs: release notes 4.6.b DOC-1791 DOC-1794 (#6417)
spectrocloud/librarium: docs/docs-content/release-notes/release-notes.md

  [Palette Virtual Clusters](../clusters/palette-virtual-clusters/palette-virtual-clusters.md#upgrade-virtual-clusters)
  page to learn how to migrate your virtual cluster workloads.

#### Improvements

- Palette and VerteX emails have improved content and updated design to ensure consistency and improve accessibility.
  The updates have been applied to sign-up, login, password resets, and billing updates emails.

#### Deprecations and Removals

> - Palette no longer supports for the edge virtual clusters. Several components no longer integrate with the
>   [libvirt](https://libvirt.org/) virtualization API, including the Palette UI, Terraform providers, the user interface,
>   and packs.
> 

Reviewer: ```suggestion
- Palette no longer supports Edge virtual clusters. Several components no longer integrate with the
  [libvirt](https://libvirt.org/) virtualization API, including the Palette UI, Terraform providers, the user interface,
  and packs.

```

### Edge

#### Features

> - <TpBadge /> Palette now allows the configuration of a primary, in-cluster registry that stores the images required for

Reviewer 2: ```suggestion
- <TpBadge /> Palette now allows the configuration of an in-cluster primary registry that stores the images required for
```
switches these adj around because *primary registry* is a term we made up to describe the architecture

>   cluster deployment. Any OCI-compliant registry can be configured as the primary registry. Refer to Deploy Cluster with

Reviewer 2: ```suggestion
  cluster deployment. Any OCI-compliant registry can be configured as the primary registry. Palette offers out-of-the-box support for Zot and Harbor registries with minimum configuration required. Refer to Deploy Cluster with
```

  Primary Registry for further information.
- The [Palette CLI](../automation/palette-cli/palette-cli.md) has a new `content` command that supports the creation of
  [content bundles](../clusters/edge/edgeforge-workflow/palette-canvos/build-content-bundle.md). This command provides
  the ability to create bundles directly from the command-line and supports operations in airgap environments. Refer to
  the [Content](../automation/palette-cli/commands/content.md) command reference page for further information.

#### Improvements

- [Local UI](../clusters/edge/local-ui/local-ui.md) has been updated with a new color scheme and logo. These new
  elements are displayed across the entire product interface. Check out the
  [Welcome to the fold: meet the new Spectro Cloud brand](https://www.spectrocloud.com/blog/meet-the-new-spectro-cloud-brand)
  blog post to learn more.

#### Deprecations and Removals

> - The `harbor-edge-native-config` pack has been deprecated. You need to use the new `registry-connect` pack to implement

Reviewer 2: ```suggestion
- The `harbor-edge-native-config` pack has been deprecated. You need to use the new `registry-connect` and `harbor` pack to implement
```

Reviewer: `registry-connect` and `harbor` packs

>   the Harbor registry. This allows you to keep your up-to-date with the latest upstream updates. Refer to Migrate from

Reviewer: ```suggestion
  the Harbor registry. This allows you to keep your up to date with the latest upstream updates. Refer to Migrate from
```
I also think there's a word missing after "your."

(FYI, "up-to-date" is an adjective, whereas "up to date" is a noun/adverb.)

  page to learn how to migrate your virtual cluster workloads.

#### Improvements

- Palette and VerteX emails have been redesigned to ensure consistency and improve accessibility. The updates have been
  applied to sign-up, login, password reset, and billing update emails.

#### Deprecations and Removals

- Palette no longer supports Edge virtual clusters. Several components no longer integrate with the
  [libvirt](https://libvirt.org/) virtualization API, including the Palette UI, Terraform providers, the user interface,
  and packs.

### Edge

#### Features

- <TpBadge /> Palette now allows the configuration of an in-cluster primary registry that stores the images required for
  cluster deployment. Any OCI-compliant registry can be configured as the primary registry. Palette offers
  out-of-the-box support for Zot and Harbor registries with minimum configuration required. Refer to Deploy Cluster with
  Primary Registry for further information.
- The [Palette CLI](../automation/palette-cli/palette-cli.md) has a new `content` command that supports the creation of
  [content bundles](../clusters/edge/edgeforge-workflow/palette-canvos/build-content-bundle.md). This command provides

>   the ability to create bundles directly from the command-line and supports operations in airgap environments. Refer to

Reviewer 3: ```suggestion
  the ability to create bundles directly from the command-line. Refer to
```
Do we support airgap? the content command itself must download data from Palette (non-airgap), but we can use it to create bundles for airgapped Edge clusters.

Author: Yes you're right.

#### Features

- The [Palette CLI](../automation/palette-cli/palette-cli.md) has a new `content` command that supports the creation of
  [content bundles](../clusters/edge/edgeforge-workflow/palette-canvos/build-content-bundle.md). This command provides
  the ability to create bundles directly from the command-line and supports operations in airgap environments. Refer to
  the [Content](../automation/palette-cli/commands/content.md) command reference page for further information.
- Terraform version 0.23.5 of the
  [Spectro Cloud Terraform provider](https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs) is
  available. For more details, refer to the Terraform provider
  [release page](https://github.com/spectrocloud/terraform-provider-spectrocloud/releases).
- A new Terraform resource, `spectrocloud_sso resource`, is now available for enforcing
  [Single Sign-On (SSO)](../user-management/saml-sso/saml-sso.md) in Palette. For more information, refer to the Spectro
  Cloud Terraform provider
  [documentation](https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs).

#### Improvements

> - The performance of Palette APIs was improved, resulting in increased [API rate limits](/api/introduction/#rate-limits)

Reviewer: ```suggestion
- The performance of Palette APIs has been improved, resulting in increased [API rate limits](/api/introduction/#rate-limits)
```

  for component events and optimized platform updates.
- The `spectrocloud_pack` Terraform data source now supports filtering based on pack type, add-on type, pack layer, and
  environment. For more information, refer to the Spectro Cloud Terraform provider
  [documentation](https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs).
- The `spectrocloud_cluster_eks` Terraform resource now supports the specification of availability zones and subnets.
  For more information, refer to the Spectro Cloud Terraform provider
  [documentation](https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs).

### Virtual Machine Orchestrator (VMO)

#### Improvements

> - The user interface and messaging for virtual machine topology has been improved to indicate the sockets, cores and

Reviewer: ```suggestion
- The user interface and messaging for virtual machine topology has been improved to indicate the sockets, cores, and
```

#### Features

- The [Palette CLI](../automation/palette-cli/palette-cli.md) has a new `content` command that supports the creation of
  [content bundles](../clusters/edge/edgeforge-workflow/palette-canvos/build-content-bundle.md). This command provides

>   the ability to create bundles directly from the command-line and supports operations in airgap environments. Refer to

Reviewer 3: ```suggestion
  the ability to create bundles directly from the command-line. Refer to
```

#### Add-on Packs

| Pack Name              | New Version |
| ---------------------- | ----------- |
| Argo CD                | 7.8.8       |
| AWS Cluster Autoscaler | 1.32.0      |
| Amazon EFS             | 2.1.6       |
| Nginx                  | 1.12.1      |
| Portworx /w Operator   | 3.2.2       |
| Spectro Proxy          | 1.5.6       |
| Registry Connect       | 0.1.0       |

#### FIPS Packs

| Pack Name                                  | New Version |
| ------------------------------------------ | ----------- |
| Amazon EBS CSI                             | 1.41.0      |
| Palette eXtended Kubernetes - Edge (PXK-E) | 1.32.2      |
| Palette Optimized RKE2                     | 1.32.2      |
| Palette Optimized RKE2                     | 1.31.6      |
| Palette Optimized RKE2                     | 1.30.10     |
| Palette Optimized RKE2                     | 1.29.14     |
| RKE2                                       | 1.32.2      |
| RKE2                                       | 1.31.6      |
| RKE2                                       | 1.30.10     |
| RKE2                                       | 1.29.14     |

#### Deprecations and Removals

> - The packs `ubuntu-libvirt`,`generic-vm-libvirt`, `centos-libvirt`, and`pfsense-vm-libvirt` have been removed. Palette

Reviewer: ```suggestion
- The packs `ubuntu-libvirt`, `generic-vm-libvirt`, `centos-libvirt`, and`pfsense-vm-libvirt` have been removed. Palette
```

>   no longer supports edge virtual clusters.

Reviewer: ```suggestion
  no longer supports Edge virtual clusters.
```

- The `harbor-edge-native-config` pack has been deprecated. You need to use the new `registry-connect` pack to implement

>   the Harbor registry. This allows you to keep your up-to-date with the latest upstream updates. Refer to Migrate from

Reviewer: ```suggestion
  the Harbor registry. This allows you to keep your up to date with the latest upstream updates. Refer to Migrate from
```
Same as earlier--missing word after "your."
